yarn.lock

```diff
@@ -0,0 +1,5980 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
```

Comment:
@fernandogcabal can you please remove this file, yarn.lock should not be inside the PR